axlearn/common/flash_attention/neuron_attention_test.py

```diff
@@ -11,6 +11,10 @@
 from axlearn.common.flash_attention.utils import mha_reference
 
 
+if jax.default_backend() != "neuron":
+    pytestmark = pytest.mark.skip(reason="Incompatible hardware, Neuron only test.")
```

Comment:
We should be more explicit with skip reason message: `"Incompatible hardware, Test only runs on AWS Neuron devices."`